WordPress/WordPressTest/PageModelTest.m

```diff
@@ -0,0 +1,62 @@
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+#import "TestContextManager.h"
+#import "Page.h"
+
+@interface PageModelTest : XCTestCase
+
+@end
+
+@implementation PageModelTest
+
+- (void)setUp {
+    [super setUp];
+}
+
+- (void)tearDown {
+    [super tearDown];
+}
+
+- (Page *)newPageForTesting
+{
+    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
+    Page *page = [NSEntityDescription insertNewObjectForEntityForName:NSStringFromClass([Page class])
```

Comment:
Ah! Thanks for the tip. I think I'll keep the context for now and maybe revisit later.